### PR TITLE
Abstract compiles

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims at modifying the visibility of functions to be able to use it within the [Abstract SDK](https://github.com/AbstractSDK/abstract/)

### CODE REVIEW

File: contracts/account/Cargo.toml [hunk 1]
```diff
@@ -7,24 +7,28 @@ edition = "2021"
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[features]
```
Comment (Kayanski) [inline contracts/account/Cargo.toml:10]: Added library feature for importing

File: contracts/account/src/auth.rs [hunk 2]
```diff
@@ -11,6 +11,10 @@ mod secp256r1;
 mod sign_arb;
 pub mod util;
 
+pub mod testing {
```
Comment (Kayanski) [inline contracts/account/src/auth.rs:14]: We need some utils for testing, they need to get re-exported

File: contracts/account/src/auth/secp256r1.rs [hunk 3]
```diff
@@ -34,23 +34,16 @@ mod tests {
...
+        .unwrap());
 
         // test with invalid msg
         let bad_value = "invalid starting msg".as_bytes();
-        let result = verify(
-            &bad_value.to_vec(),
-            signature_bytes.as_slice(),
-            &verifying_key_binary,
-        );
+        let result = verify(bad_value, signature_bytes.as_slice(), &verifying_key_binary);
```
Comment (Kayanski) [inline contracts/account/src/auth/secp256r1.rs:46]: Clippy

File: contracts/account/src/auth/sign_arb.rs [hunk 4]
```diff
@@ -19,7 +19,7 @@ pub fn verify(
     Ok(verification)
 }
 
-fn wrap_message(msg_bytes: &[u8], signer: Addr) -> Vec<u8> {
+pub fn wrap_message(msg_bytes: &[u8], signer: Addr) -> Vec<u8> {
```
Comment (Kayanski) [inline contracts/account/src/auth/sign_arb.rs:22]: Need that for testing

File: contracts/account/src/execute.rs [hunk 5]
```diff
@@ -87,7 +87,7 @@ pub fn after_tx() -> ContractResult<Response> {
 
 pub fn add_auth_method(
     deps: DepsMut,
-    env: Env,
+    env: &Env,
```
Comment (Kayanski) [inline contracts/account/src/execute.rs:90]: Env ownership is not needed here

**Review by ash-burnt — APPROVED** [review]

**Review by ash-burnt — APPROVED** [review]

**Pull request merged**